pinot-controller/src/main/java/com/linkedin/pinot/controller/api/restlet/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -17,6 +17,7 @@
 
 import com.alibaba.fastjson.JSONObject;
 import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
That requires a full clean up because fasterxml and codehaus jackson are not compatible with each other...